handlers.go

```diff
@@ -967,6 +967,7 @@ func (s *Server) redirectContentAdding(c echo.Context, u *User) error {
 		log.Warnf("failed to get preferred upload endpoints: %s", err)
 		return err
 	} else if len(uep) > 0 {
+		//#nosec G404 - crypto/rand it's not necessary for this use case
```

Comment:
what is the issue here?

Reply:
Hello,

the issue here is that the gosec considers the rand.Intn() insecure for generating random numbers:

![image](https://user-images.githubusercontent.com/12394082/181274038-08316025-7ceb-48c3-a754-13845d23cab8.png)

but in this line, rand.Intn() is used to get a number between [0, len(uep)] so there is no need for a truly secure random number.

Reply:
Agreed @mvjq - we should be using common sense with these flags, this is fine.